input/input.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://utilities91.netlify.app/styles.css" />
+    <link rel="stylesheet" href="./input.css" />
+    <title>Input</title>
+</head>
+
+<body>
+    <div>
+        <label for="name">Name</label>
+        <input class="py-1 px-2 text-base border rounded-sm" id="name" type="text" placeholder="Enter your name">
+    </div>
+
+
+    <div>
+        <label for="password">Name</label>
+        <input class="py-1 px-2 text-base border rounded-sm" id="password" type="password"
+            placeholder="Enter your password">
+        <span class="success-msg p-1 rounded-sm red">Login successfull!</span>
+    </div>
+
+    <div>
+        <label for="email">Name</label>
+        <input class="py-1 px-2 text-base border rounded-sm" id="email" type="email" placeholder="Enter your email">

```

Comment:
label for email and you mentioned label content as Name.. is it intentional? If yes ignore the comment.